### PR TITLE
FW-56 Add skynet.yaml

### PR DESCRIPTION
**Pull request by wesleybalvanz-wf**
This runs the unit tests inside skynet to satisfy the release pipeline QA requirement.

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: run_tests.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ex
+
+RESULT=0
+
+filename="test-reports/test_output"
+
+pub get
+
+pub run test --file-reporter=json:$filename.json || RESULT=1
```
[the remark this answers is not in the document]
Comment (wesleybalvanz-wf) [reply run_tests.sh]: My bad of course, updated.

**Review by evanweible-wf — APPROVED** [review]

**Comment by evanweible-wf** [comment]
QA +1
- [x] CI passes
- [x] Skynet is running unit tests
- [x] Unit test results show up in Skynet

**Comment by wesleybalvanz-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**